### DISCUSSION TITLE
Auto Upload Hero 5 - External Power “0%” Battery

### DISCUSSION THREAD

**Discussion by standpointmedia** [gopro/labs#580] [Q&A]
Hi there, I'm trying to work out how to get my Hero 5 to auto upload during a hour long Wi-Fi window. The Hero 5 will be powered externally by USB battery (via a BlinkX Timelapse controller).

I keep getting an error right now saying "connect to power..etc"

I am assuming that the error is because the camera is reading the battery level as 0%, and auto upload requires at least 40%?

Hopefully there is a way around this?

Thanks in advance for your help!

**Comment by dnewman-gpsw** [comment]
Confused, this camera model doesn't have auto upload, no does it support Labs.

**Reply by standpointmedia** [reply]
According to the minimum requirements for Auto Upload, the Hero 5 is supported?

Can see now that Labs only supports the Hero 5 Session, but not the main Hero 5, weird.

![image](https://github.com/gopro/labs/assets/38859946/5709d7cc-bf73-4fa0-9ced-b67fe79129d2)

**Reply by dnewman-gpsw** [reply]
Cool, I haven't kept up with HERO5 features as it is not part of Labs. Nice that it has auto upload, I really had no idea. HERO5 Session doesn't have full Labs, it is barely 10% of the features, it was only created to fix a bug that made sense at the time. Labs started with HERO8, with the newer features only on newer models.  Before you ask, there will not be Labs for HERO5 or 6, or any more Labs updates for 7, 8 or 9.  MAX, 10, 11 and 12, are the current Labs cameras.
